src/index.ts

```diff
@@ -16,6 +20,11 @@ export const Config: Schema<Config> = Schema.intersect([
       Schema.array(String),
       Schema.transform(String, (prefix) => [prefix]),
     ] as const).description('使用特定前缀触发对话。').default(['!', '！']),
+    conversationContext: Schema.union([
+      Schema.const('user' as const).description('用户独立'),
+      Schema.const('channel' as const).description('频道独立'),
+      Schema.const('both' as const).description('频道内用户独立'),
+    ]).description('对话历史上下文缓存方式。').default('channel'),
```

Comment:
Are there better naming and/or better translations?

Reply:
That's too loooooong.

I'd like a shorter one.

Reply:
Well, in my opinion, it would be better to focus on "sharing a context" rather than "isolate a context". So my suggestion is "contextInteraction".

Reply:
Can you provide a translation of it in Chinese as well?

Reply:
I think it's a good choice to use "上下文共享模式(or 方式)"
If you think that it would be better to focus on "sharing a context" rather than "isolate a context" . It's better to change the transaction of options from '独立' to '共享' . It's even better to change from "频道独立" to "频道内独立"

Reply:
```suggestion
    conversationContext: Schema.union([
      Schema.const('user' as const).description('用户共享'),
      Schema.const('channel' as const).description('频道共享'),
      Schema.const('both' as const).description('频道内用户独立'),
    ]).description('上下文共享方式。').default('channel'),
```

See if this satisfy yours?

Reply:
I think we can simply use `interaction` since there is no ambiguity.

Reply:
Well, tasikani. Since we have description for every configuration items, it should be good.

Reply:
上下文共享方式：

- 用户上下文
- 频道上下文
- 频道+用户上下文

要不要这样改？

Reply:
I think the first two phases make sense, but the last one might baffles people?